### ISSUE TITLE
googleads examples: if 'entries' in page check is not suitable to check for results

### ISSUE THREAD

**Issue by gitlech** [googleads/googleads-python-lib#284]
in many of the googleads examples, eg. get_campaigns.py there is a 

```python
if 'entries' in page:
    print(something)
else:
    print('No campaigns were found.')
```

However, the result of a request not finding data looks like:
    {
        'totalNumEntries': 0,
        'Page.Type': 'CampaignPage',
        'entries': []
    }

So there is always an 'entries' key. and else is never reached.
It would be better to have instead:
```python
if page['totalNumEntries'] > 0:
```

**Comment by msaniscalchi** [comment]
Hello,

Thanks for the report! That's true, and looking over the samples, it seems to be a pattern used in many of them. While I don't expect many samples to raise an exception or otherwise fail as a result of this, it won't actually indicate cases where none of the given resource type were found, which is certainly undesirable. Sure, we'll look into fixing this.

Regards,
Mark

**Comment by grivescorbett** [comment]
Mark I just noticed this myself somewhere else, I think it's a suds/zeep thing.

**Comment by grivescorbett** [comment]
Somewhat addressed in 12.2.0 for samples that run forever.
